cookiecutter.json

```diff
@@ -2,8 +2,8 @@
   "project_name": "",
   "project_slug": "{{ cookiecutter.project_name|lower|replace(' ', '')|replace('-', '_') }}",
   "project_dash_slug": "{{ cookiecutter.project_slug|replace('_', '-') }}",
-  "owner_email": "",
-  "ssh_key_fingerprint": "",
+  "owner_email": "{{ cookiecutter.owner_email }}",
+  "ssh_key_fingerprint": "{{ cookiecutter.ssh_key_fingerprint }}",
```

Comment:
You had these correct before. The value here is the default, previously an empty string. If you pass in a value it overwrites it. `cookiecutter.owner_email` is a reference to itself so it won't set it to anything different. I'm actually not sure what it'll do if you set it to itself.